tests/compiler_tests.rs

```diff
@@ -175,6 +175,24 @@ fn it_compiles_plus_operator() {
     compiler.compile().unwrap()
 }
 
+#[test]
+#[should_panic]
+fn it_not_compile_plus_operator() {
```

Comment:
Should be "it_does_not", instead of "it_not". I think it would be even better to call it "it_panics" to match the assertion.

Reply:
hmm, i can call it "it_panics_on_binay_plus" because in this case plus taking two arguments.

Reply:
right, lets call it ```it_panics_when_adding_numeric_to_string``` then :)

Reply:
I just use macros also in the compiler test and create one it functions, take a look ;p